### PR TITLE
Refresh jazzy documentation

### PR DESCRIPTION
**Pull request by djones6**

### CODE REVIEW

**Review by helenmasters — CHANGES_REQUESTED** [review]
I think it would be good to make a few improvements to the jazzy doc whilst we're in here!

The description is currently "An implementation of Kitura’s TemplateEngine protocol. In particular this templating engine takes files in Markdown (.md) format and converts them to HTML. In addition this class has some helper methods for taking Markdown formatted text and converting it to HTML.

I think it'd be much clearer to make it something like "An implementation of Kitura’s TemplateEngine protocol that enables a Kitura server to serve HTML content generated from Markdown templates (`.md` files). This class also has some helper methods that take Markdown formatted text from a Data struct, or String and convert it to HTML."

**Comment by helenmasters** [comment]
fileExtension - The file extension of files in the views directory that will be rendered by a particular templating engine.

I think this should be "The file extension of files that will be rendered by the KituraMarkdown template engine." Perhaps we should also add "By default Kitura will search for these in the  ./Views/ directory, you can customize this by setting the `router.viewsPath` property."

**Comment by helenmasters** [comment]
Within init() - Create a KituraMarkdown instance needs a full stop on the end.

**Comment by helenmasters** [comment]
There is a parameter called `options` which appears in a couple of places which needs a sentence next to it rather than "markdown options".

**Comment by helenmasters** [comment]
Every `from` parameter (there are 3 references to this parameter) needs a full stop at the end of the associated description.

**Comment by helenmasters** [comment]
The `pageTemplate` parameter has the description "The HTML page template used" which needs a full stop at the end.

**Comment by helenmasters** [comment]
There is no difference in the descriptions for:
`public static func render(from: String) -> String` and `public static func render(from: String, pageTemplate: String) -> String` I'd like there to be a difference as otherwise why are we passing in pageTemplate?

**Comment by helenmasters** [comment]
The API doc for the struct `MarkdownOptions` could do with some descriptions adding and full stops at the end of sentences and parameter descriptions.

**Comment by helenmasters** [comment]
It would be good to add at least one usage example, there is one in the github README for this repo if required.

**Comment by djones6** [comment]
I've addressed some of the review comments.  Still need:
- usage example
- some kind of example of how the HTML page template should be overridden (the sole purpose of `MarkdownOptions`).  This doesn't seem to be covered in any of our guides so I have inferred its usage from the code.

**Comment by djones6** [comment]
@helenmasters Review comments addressed.  And I've also (hopefully) solved the jazzy-doc mergeability issue.

File: Sources/KituraMarkdown/KituraMarkdown.swift [hunk 1]
```diff
@@ -19,46 +19,87 @@ import Foundation
...
+ page template. The page template should contain
+ `<snippetInsertLocation></snippetInsertLocation>`, which will be substituted with the
+ generated HTML content.
+
+ _Note: If you do not wish to customize the page template, `"default"` can be specified._
+
+ ### Usage Example:
+
+ ```swift
+ let markdownOptions = MarkdownOptions(pageTemplate: "<!DOCTYPE html><html lang=\"en\"><head><meta charset=\"UTF-8\"></head><body><div><snippetInsertLocation></snippetInsertLocation></div></body></html>"))
```
Comment (helenmasters) [inline Sources/KituraMarkdown/KituraMarkdown.swift]: If you remove the extra ')' at the end of this line the PR looks good to go.

**Review by helenmasters — APPROVED** [review]

**Pull request merged**